docs/facebook-instant-articles.md

```diff
@@ -0,0 +1,40 @@
+# Facebook Instant Articles
+
+Facebook Instant Articles enables publishers to monetise and make content available natively within Facebook's website and apps using a custom feed.
+
+This component is powered by the [official Facebook Instant Articles plugin](https://github.com/humanmade/facebook-instant-articles-wp).
+
+## Pre-requisites
+
+* A Facebook page for your organisation
+* A Facebook App
+
+## Configuration
+
+Configuration is managed via the CMS admin interface. Follow the wizard to complete the setup before moving onto customising how your content will be transformed and displayed.
```

Comment:
Could we have this config via our config file? Seems it would be much better suited there.

Reply:
Yeah I think so, there's quite a few options on the wizard but I think we can. It's likely we'll need to look at supporting this too https://github.com/humanmade/platform-core/issues/8